lighthouse/lighthouserc.js

```diff
@@ -59,6 +61,23 @@ const AUDIT_URLS = {
       'https://www.test.bbc.com/pashto/afghanistan-52643309?renderer_env=live',
     ],
   },
+
+  TOPIC_PAGE: {
+    live: ['https://www.bbc.com/pidgin/topics/c0823e52dd0t'],
+    test: ['https://www.test.bbc.com/pidgin/topics/c0823e52dd0t'],
```

Comment:
Test topic points to live environment, so the content should be the same as on live